### PR TITLE
Update FSF address

### PR DESCRIPTION
**Pull request by tamplan**
Update [FSF address](https://www.fsf.org/about/contact/)

### CODE REVIEW

**Comment by raveit65** [comment]
Correct Address can be found here https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
`51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA`
But in commit it looks like this:
```
- *  Foundation, Inc., 59 Temple Street #330, Boston, MA 02110-1301, USA.
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```
@lukefromdc @vkareh 
I don't live in the states but Is there a decimal point missing after 021101301, when it is written in one line?
Or  can we accept this?
Personal i don't care if there is a point or not.

**Comment by raveit65** [comment]
Thanks for fixing that.

File: src/actions.h [hunk 1]
```diff
@@ -17,7 +17,11 @@
  *
  *  You should have received a copy of the GNU General Public License
  *  along with this program; if not, write to the Free Software
- *  Foundation, Inc., 59 Temple Street #330, Boston, MA 02110-1301, USA.
+<<<<<<< HEAD
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+=======
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+>>>>>>> master
```
Comment (vkareh) [inline src/actions.h]: Hmmm... there seems to have been a merge conflict, and now the markers are all over the place. This needs to be fixed.

**Comment by raveit65** [comment]
@tamplan 
I tried to fix this merge conflict, but nearly ever file needs to be edit by hand. Sorry, you really like to lean git commands at command line.
Can you please make a new PR from a new repo?

**Comment by tamplan** [comment]
> @tamplan
> I tried to fix this merge conflict, but nearly every file needs to be edit by hand. Sorry, you really need to learn git commands at command line.
> Can you please make a new PR from a new repo?

Yes, no problem, i make a new repo.

Regards

**Pull request closed without merging**